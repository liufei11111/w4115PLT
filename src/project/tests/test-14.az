Int main(Int argc, String argv){
Int a;
Float b;
String c;
Matrix m(1,2);
Option o = {};
Structure s = {};
Boolean boo;
Void v;
}