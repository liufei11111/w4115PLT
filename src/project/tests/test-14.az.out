
public static int main(int argc, String argv)
{
try{int a=0;
double b=0.0;
String c=null;
Matrix m= new Matrix(1, 2);
Option  o = new Option();
;
Structure s = new Structure();
;
boolean b=false;
void v;
}catch(Exception e){return -1;}}

